app/src/main/kotlin/com/wire/android/ui/authentication/create/details/CreateAccountDetailsScreen.kt

```diff
@@ -127,174 +125,179 @@ private fun DetailsContent(
-                        .padding(
-                            horizontal = MaterialTheme.wireDimensions.spacing16x,
-                            vertical = MaterialTheme.wireDimensions.spacing24x
+                item {
+                    Text(
+                        text = stringResource(R.string.create_personal_account_details_text),
+                        style = MaterialTheme.wireTypography.body01,
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .padding(
+                                horizontal = MaterialTheme.wireDimensions.spacing16x,
+                                vertical = MaterialTheme.wireDimensions.spacing24x
+                            )
+                    )
+                }
+
+                item {
+                    WireTextField(
+                        value = state.firstName,
+                        onValueChange = onFirstNameChange,
+                        placeholderText = stringResource(R.string.create_account_details_first_name_placeholder),
+                        labelText = stringResource(R.string.create_account_details_first_name_label),
+                        labelMandatoryIcon = true,
+                        state = WireTextFieldState.Default,
+                        keyboardOptions = keyboardOptions,
+                        modifier = Modifier
+                            .padding(
+                                start = MaterialTheme.wireDimensions.spacing16x,
+                                end = MaterialTheme.wireDimensions.spacing16x,
+                                bottom = MaterialTheme.wireDimensions.spacing16x
+                            )
+                            .focusRequester(focusRequesterFirstName)
+                            .testTag("firstName"),
+                    )
+                }
+
+                item {
+                    WireTextField(
+                        value = state.lastName,
+                        onValueChange = onLastNameChange,
+                        placeholderText = stringResource(R.string.create_account_details_last_name_placeholder),
+                        labelText = stringResource(R.string.create_account_details_last_name_label),
+                        labelMandatoryIcon = true,
+                        state = WireTextFieldState.Default,
+                        keyboardOptions = keyboardOptions,
+                        modifier = Modifier
+                            .padding(
+                                start = MaterialTheme.wireDimensions.spacing16x,
+                                end = MaterialTheme.wireDimensions.spacing16x,
+                                bottom = MaterialTheme.wireDimensions.spacing16x
+                            )
+                            .focusRequester(focusRequesterLastName)
+                            .testTag("lastName"),
+                        shouldDetectTaps = true,
+                        onTap = {
```

Comment:
This will work when user taps on the specific input, but what if he uses the ImeAction "Next" button on the keyboard?

Reply:
The aim of this PR is to make user aware that there are more fields to fill, if he is using ImeAction, then he will finish sign up process 🙂

Reply:
Ok, I can understand that, but the user can always click "Next" ime action only as long as he sees the inputs that are empty and if he fills all the inputs that he sees (for instance first name, last name, team name and password) he can close the keyboard thinking that it should be all, which is exactly what this ticket is all about 😅 The suggestion would be to maybe scroll after the input gets the focus, then every time we change the input, no matter which way, we have this input being scrolled up so we can see that there are also items below.

Reply:
That's the idea but I didn't find a way to do it. If you know how to make it feel free to update the PR